spec/std/concurrent/select_spec.cr

```diff
@@ -191,7 +191,7 @@ describe "select" do
     x.should eq 2
   end
 
-  it "stress select with send/receive in multiple fibers" do
+  pending_win32 "stress select with send/receive in multiple fibers" do
```

Comment:
FTR in theory nothing is preventing this to work, all of the ingredients are there, but it's still failing.